libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/util/CompositeMapboxNavigationObserver.kt

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.navigation.dropin.util
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+internal class CompositeMapboxNavigationObserver(
+    vararg elements: MapboxNavigationObserver
+) : MapboxNavigationObserver {
+    private val observers = mutableListOf(*elements)
+
+    fun add(o: MapboxNavigationObserver) = observers.add(o)
+    fun remove(o: MapboxNavigationObserver) = observers.remove(o)
```

Comment:
These should call `onAttached` and `onDetached`.

Do we use these functions?

Reply:
No.
Yes.